src/lib/service-bus/RedisStream.js

```diff
@@ -0,0 +1,23 @@
+/**
+ *
+ */
+export class RedisStream {
```

Comment:
Is this used, are we not going for the emulator instead?

Reply:
This is WIP, it's still there as a placeholder for the other solution.